packages/cli/src/commands/connector/utils.ts

```diff
@@ -144,6 +144,11 @@ const officialConnectorPrefix = '@logto/connector-';
 
 type PackageMeta = { name: string; scope: string; version: string };
 
+// NPM registry text search returns too many irrelevant results 1000+.
+// Making it impossible to filter out all official connectors in one go.
+// So we added an additional `maintainer` filter to reduce the number of irrelevant results.
+const maintainer = 'gaosun';
```

Comment:
should be `scope:logto`?

Reply:
```ts
const officialScope = 'logto';
const officialConnectorPrefix = `@${officialScope}/connector-`;
```

Reply:
discussed offline, add some comments here. `scope` in the NPM search API doesn't work as expected.